client/test/e2e/debtors/debtorgroups.spec.js

```diff
@@ -0,0 +1,65 @@
+/* global browser, element, by, protractor */
+var chai = require('chai');
+var expect = chai.expect;
+
+var helpers = require('../shared/helpers');
+helpers.configure(chai);
+
+var components = require('../shared/components');
+var FU = require('../shared/FormUtils');
+
+describe('Debtor Groups Management', function () {
+  'use strict';
+
+
+  /** @const */
+  var root = '#/debtors/groups';
+
+  before(function () { browser.get(root); });
+
+  it('lists base test debtor groups', function () {
+
+    expect(element.all(by.css('[data-group-entry]')).count()).to.eventually.equal(2);
```

Comment:
Agreed! As you mentioned here https://github.com/IMA-WorldHealth/bhima-2.X/pull/364/files/7d8cacc277c146493e7d26f8542dd1c923b9149a#r61517367 this could be modelled as a component. If this was the case it could be represented here by a shared component wrapper.